Src/BootCamp.Chapter/Program.cs

```diff
@@ -1,12 +1,48 @@
-﻿using System;
-
-namespace BootCamp.Chapter
-{
-    class Program
-    {
-        static void Main(string[] args)
-        {
-            Console.WriteLine("Hello World!");
-        }
-    }
-}
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            string name, surname, ageInput, weightInput, heightInput;
+            int age;
+            double weight, height, bodyMassIndex;
+
+            Console.Write("Name: ");
+            name = Console.ReadLine();
+
+            Console.Write("Surname: ");
+            surname = Console.ReadLine();
+
+            Console.Write("Age: ");
+            ageInput = Console.ReadLine();
+
+            Console.Write("Weight: ");
+            weightInput = Console.ReadLine();
+
+            Console.Write("Height: ");
+            heightInput = Console.ReadLine();
+
+            age = int.Parse(ageInput);
+            // Convert age string input weight into int

```

Comment:
Comments are good, but normally the convention is to put them above (and not below) the line that you explain. They are meant to explain, so it is nicer to read them before reading the actual line of code.